### PR TITLE
APPIOS-2873: Parcels: Create SDK

### PR DESCRIPTION
**Pull request by softwaresaiyajin**

### CODE REVIEW

**Comment by softwaresaiyajin** [comment]
Tests passed
Screen Shot 2021-04-26 at 10 33 56 AM

File: Sources/PayseraParcelSDK/Entities/PSSize.swift [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import ObjectMapper
+
+public final class PSSize: Mappable {
+    
+    public var name: String!
+    public var height: Float!
```
Comment (Pattio) [inline Sources/PayseraParcelSDK/Entities/PSSize.swift:6]: Let's use Double (same for values below)

File: Sources/PayseraParcelSDK/Entities/PSTerminal.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+public final class PSTerminal: Mappable {
+    
+    public var id: String!
+    public var country: String!
+    public var postalCode: String!
+    public var number: String!
+    public var city: String!
+    public var address: String!
+    public var workingHours: String?
+    public var latitude: Float?
```
Comment (Pattio) [inline Sources/PayseraParcelSDK/Entities/PSTerminal.swift:12]: Double

File: Sources/PayseraParcelSDK/Entities/PSTerminal.swift [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    public var longitude: Float?
```
Comment (Pattio) [inline Sources/PayseraParcelSDK/Entities/PSTerminal.swift:13]: Double

File: Sources/PayseraParcelSDK/Services/ParcelAPIRequestRouter.swift [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+        self.method = method
+        self.path = path
+        self.parameters = parameters
+    }
+}
+
+enum ParcelAPIRequestRouter {
+    
+    private static let baseURL = URL(string: "https://parcel-api.paysera.net/public/rest/v1")!
+
```
Comment (Pattio) [inline Sources/PayseraParcelSDK/Services/ParcelAPIRequestRouter.swift:24]: Let's add `MARK` with method type so it's easier to add methods later just by glancing and mark header

**Review by Pattio — CHANGES_REQUESTED** [review]

File: Sources/PayseraParcelSDK/Services/ParcelAPIClient.swift [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    
+    public func getCountries() -> Promise<PSMetadataAwareResponse<PSPackageCountry>> {
+        doRequest(requestRouter: ParcelAPIRequestRouter.getCountries)
+    }
+    
+    public func getCities(countryCode: String) -> Promise<PSMetadataAwareResponse<PSPackageCity>> {
+        doRequest(requestRouter: ParcelAPIRequestRouter.getCities(countryCode: countryCode))
+    }
+    
+    public func registerParcel(payload: PSPackage, payOnReceive: Bool) -> Promise<PSPackage> {
```
Comment (Pattio) [inline Sources/PayseraParcelSDK/Services/ParcelAPIClient.swift:46]: `Parcel` -> `Package` ? (same few instance below)

File: Sources/PayseraParcelSDK/Entities/Filter/PSPackagePriceFilter.swift [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import ObjectMapper
+import PayseraCommonSDK
```
Comment (Pattio) [inline Sources/PayseraParcelSDK/Entities/Filter/PSPackagePriceFilter.swift:2]: Not needed?

**Review by Pattio — CHANGES_REQUESTED** [review]

**Pull request merged**